src/Rector/v9/v0/DatabaseConnectionToDbalRector.php

```diff
@@ -22,11 +26,14 @@ final class DatabaseConnectionToDbalRector extends AbstractRector
     private $typo3NodeResolver;
 
     /**
-     * @noRector
+     * @var DatabaseConnectionToDbalRefactoring[]
      */
     private $databaseConnectionRefactorings;
 
-    public function __construct(Typo3NodeResolver $typo3NodeResolver, iterable $databaseConnectionRefactorings)
+    /**
+     * @param DatabaseConnectionToDbalRefactoring[] $databaseConnectionRefactorings
+     */
+    public function __construct(Typo3NodeResolver $typo3NodeResolver, array $databaseConnectionRefactorings)
```

Comment:
The parameter autowiring can be simplified thaks to Symplify :)

https://tomasvotruba.com/blog/2021/01/18/4-ways-symplify-makes-working-with-symfony-more-fun/#3-autowire-list-of-services